### ISSUE TITLE
EPIC: Main Room Layout

### ISSUE THREAD

**Issue by nelsonic** [bmhco/shared#4]
@Stephanymtr please share the measurements you've collected for the main room of @home 🏡 
and any layout iterations you've worked on so we can capture & discuss. 🙏

**Comment by nelsonic** [comment]
Sssiiiiiiiiiiiiiiiiiiiiiccckk!!! 😍 `#want!`
Ok. this is an excellent mood board! ❤️ 
Please let's get some of these photos printed. 🖨️ 
Where are we going to _start_? 💭 

CC: @iteles 🔔

**Comment by Stephanymtr** [comment]
@nelsonic I printed everything out. Thank you for your help. Where would you like me to set up the mood board ? ![image](https://user-images.githubusercontent.com/91985721/205287181-2441eb74-c5c1-48d3-a447-9d18652bd873.jpeg)

**Comment by nelsonic** [comment]
Good question. I think we should put up the pictures in situ today. 💭

**Comment by nelsonic** [comment]
@iteles do you know where we have blue/white tac to _temporarily_ put these up 
**or** should we use our non-perm double-sided tape? 💭

**Comment by Stephanymtr** [comment]
@nelsonic, @iteles and I briefly talked about my suggestions for the layout. Below are a few points that I remember:

-> Switch current art/music area with math, language and culture area because it's sunnier on the west side of the room. 
-> Music area needs to be bigger.
-> Include more seating, so that all children have chairs.
-> Display art supplies as activities on trays.
-> Link the Movement/Fun area to the north instead of the west side of the room, as I suggested previously: 
> "...we could even make it into a monkey bar that starts off as a climbing wall next to the music area, goes over the doorway as a monkey bar and down where the pikler triangle is as a pole or a rope..."

If you guys remember anything else we talked (regarding the layout), please share. @nelsonic verbally mentioned a couple of fun ideas he had for the space, so if you want to share with us, please do. We can gather all of our thoughts in one place and start working on this.

**Comment by nelsonic** [comment]
@Stephanymtr this is a good summary of our conversation. 👍 
We need to plan for the _potential_ numbers we want to accomodate 
and thus the 3 chairs we have might not be "enough" ...
They are certainly "enough" in the short-term #7 
But this layout/planning exercise is to maximise the available space for the longer term.

**Comment by Stephanymtr** [comment]
- We could have a large round a table for team work activities and small tables with single seats for independent work. However, I believe that bulky furniture that only serves one purpose goes against  "maximise the available space for the longer term" plan. In fact, I'd suggest avoiding bulky furniture unless it's not taking up floor space. 

![image](https://user-images.githubusercontent.com/91985721/206676864-f2bef760-7e61-430a-881d-4251f505726a.png)

- I was thinking that we could use small mobile desks that are easy to rearrange and can be used for solo learning or can be configured to encourage collaboration. There are certain seating that stacks and flip top tables that nest together when they are not being used. Would this be too tacky ? A1's learning tower turns into a desk and that's pretty cool! 

![image](https://user-images.githubusercontent.com/91985721/206676969-2daa7796-a4da-4284-842f-f3e077042f7c.png)
![image](https://user-images.githubusercontent.com/91985721/206677109-01511dec-b2c2-412c-8059-9975f4824d00.png)

- I even thought that we could create an elevated seating area above the floor. It's a fun idea, but not very practical for younger children who will be learning to carry their activities in tray from the shelf to the desk and back.

**Comment by nelsonic** [comment]
All solid ideas. I'm a huge fan of multi-level/tiered room as it expands the imagination of the children to think _vertically_! 🗼

**Comment by Stephanymtr** [comment]
@iteles The changes we did worked well last year, but A2 has outgrow her NIDO area. She is definitely an explorer! The ballet corner, ball pit and the trampoline seem to be her current favourite places. As for A1, he enjoys reading books and playing games in the ball pit. You can also find him in the there playing golf, rugby, football, baseball, .... ⚾ 🏈 🏐  ball games! However, the kiddos haven’t really been taking full advantage of all the activities being offered in the playroom. I personally don’t think changing the layout now is as important as improving the systems and strategies currently used to keep up with the activities presented to the two kiddos. When the new system is implemented, then the layout can be catered to the new children that will be coming. 

**Layout suggestions:**
What are the ages of the children you will be welcoming in the space? I can make a few suggestions on the current layout, but I can only do it based off on the ages of the 2 children who are using the space atm. 
- If you could connect the NIDO area to the ballet corner, A2’s space would grow double in size. This would hopefully give her a well deserved space upgrade. I also feel that this would separate her area a bit more and should hopefully become more clear which toys are for her and which toys are for A1.
- A1 has certain toys that are quite bulky and some of them are not being used. Given this room is supposed to be intentionally-designed to optimise his opportunities to learn through play, I think these bulky items (for ex, cardboard house) need to go 🙈 
- I like how almost half of the room has an open space. Children should have space to run around and dance and use their bodies without bumping into hard furniture or knocking things over. This area gets used so often especially during raining days. My only comment here would be to move the musical instruments to another location because they are currently “stored” in this open space. Implementing the musical instrument furniture Gui designed would protect the instruments and the kids’ toes when running around.  Every time a ball (or a golf swing, or a hockey stick etc) hits the keyboard, my heart breaks a little 🥲 

 **Systems and strategies:**
- Implementing a new toy rotation strategy should encourage the kiddos to do a bit more sit down work/activities. Prioritising toy rotation will keep the children curious. This will also encourage children to move about independently and comprehend their options without feeling overwhelmed by too many options or feeling uninterested by the same options.
- Making sure all the toy categories are present and properly displayed in the space should help with optimising children’s opportunities to learn through play. One of the current obstacles that I see atm is the art material being put away in that mobile cart. It worked well before, but I feel that because the materials aren't being displayed on the shelf, A1 doesn't show interest in doing them as often. It also doesn’t give him the opportunity to be independent and pick out art/craft related materials on his own. He needs adult help to set it up... especially now that I moved it to the hallway 😥 (_A2 kept reaching for the wheels and her little fingers kept getting stuck_)
- The new shelving units installed were great to display more activities, but A2 can now reach for all of them and this limits A1’s access to his age appropriate activities.  Having a strategy to go about this issue is really important because otherwise (once the new toy rotation system is implemented) all the toys he will play with **will have to be** NIDO friendly. 
- Another small comment here is the current tall shelf unit that has been used to store extra toys and other stuff. It doesn’t get used to display toys, but even if did, it’s still quite tall for A1. I also noticed he tried to climb it a few time trying to reach something (ex jumping cord) that was placed up high on purpose. The unit isn’t attached to the wall so that also scares me a bit.

**Comment by Stephanymtr** [comment]
@iteles In case we don't have time to chat about it... as requested, here are a few changes/suggestions I thought of while playing with Ami. 


-Have an open space for floor activities 
-More kids=more chairs
-Move rug & trampoline 
-Remove broken mirror and install new ones in the ballet corner 
-Remove all hard balls from the space because if they hit it too hard, they could break break things/hurt others. Soft ball need to be placed at a strategic place so that little ones are not tempted to bite/eat it.
-Remove the tall shelving until OR attach it to the wall leaving a learning tower nearby for the older ones to be able to reach their toys without climbing the unit and potentially hurting themselves. 
-Set up the music area in a way that instruments are safe when put away as opposed to being “put away” on the floor. 
-Treat all the arts and craft as toys and rotate them leaving a couple of safe options for all ages available/within reach displayed on trays. 
-The art section should also have prep materials like cloths (for wiping spills), little containers (for water) hooks ( for bibs that children can put on easily) etc

**Comment by iteles** [comment]
@nelsonic Do you know if this exists on any device in an editable format?

**Comment by nelsonic** [comment]
@iteles nope. but we can replicate it together quick-a-lee in Google Docs (Draw) or InkScape if you have `T25m`. 👩‍💻
